tests/core/test_integration.py

```diff
@@ -2337,12 +2337,6 @@ def test_multi(mocker):
     assert len(plan.new_snapshots) == 4
     context.apply(plan)
 
-    context = Context(
-        paths=["examples/multi/repo_1"],
-        engine_adapter=context.engine_adapter,
-        state_sync=context.state_sync,
-        gateway="memory",
-    )
```

Comment:
Why was this removed?

Reply:
revised it to take the previous engine_adapter and added it back